tools/fill_conf_values.py

```diff
@@ -30,7 +50,8 @@ def fill_config_file_values(template_paths):
             env = jinja2.Environment(
                 loader=jinja2.FileSystemLoader(tpath),
             )
-            env.filters['md5sum'] = md5sum
+            for (name, func) in custom_filters.items():
+                env.filters[name] = func
```

Comment:
```suggestion
            env.filters.update(custom_filters)
```
Any reason not to use this simpler form?

Reply:
No, that's good!